source/connectors/kafka-installation.txt

```diff
@@ -0,0 +1,35 @@
+.. _kafka-installation:
+
+===============================
+Install MongoDB Kafka Connector
+===============================
+
+.. default-domain:: mongodb
+
+Install with Confluent Hub Client (Recommended)
+-----------------------------------------------
+
+1. Install the `Confluent Hub Client <https://docs.confluent.io/current/connect/managing/confluent-hub/client.html>`_
+   if necessary.
+
+2. Install the `MongoDB Connector for Apache Kafka <https://www.confluent.io/hub/mongodb/kafka-connect-mongodb>`_
+   component using the Confluent Hub Client.
+
+Install Manually
+----------------
+
+If you require an alternative installation method, the connector release
```

Comment:
Sure. One unrelated planned change to this section I wanted to mention to avoid a round trip question-explanation is to omit the snapshots since I am not certain we have snapshots of releases (only dev versions) on sonatype and I have not gotten any response from Ross Lawley who I think is responsible for maintenance. This should simplify this manual installation instructions.

Reply:
Seems reasonable 👍